staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -185,6 +191,20 @@ func (t *tracker) List(gvr schema.GroupVersionResource, gvk schema.GroupVersionK
 	return list.DeepCopyObject(), nil
 }
 
+func (t *tracker) Watch(gvr schema.GroupVersionResource, ns string) (watch.Interface, error) {
+	t.lock.Lock()
+	defer t.lock.Unlock()
+	fakewatcher := watch.NewFake()
+	if _, exists := t.watchers[gvr]; !exists {
+		t.watchers[gvr] = make(map[string]*watch.FakeWatcher)
```

Comment:
Is there a way to make a buffered fake watcher?  Since its just for testing, I'm fine with the idea of making a reasonably sized buffer and saying their watcher has to consume within 100-ish messages